Ongaku/ViewController.swift

```diff
@@ -124,7 +126,7 @@ class ViewController: NSViewController {
                 break
             default:
                 presence.details = "Music is most likely closed."
-                presence.state = "If so, please quit this app. If not, please file a bug."
+                presence.state = "If so, please quit this app. If not, please file a bug."   
```

Comment:
I'll merge this on the condition you fix other things later and fix this whitespace as it is going to bug me